### DISCUSSION TITLE
Add Next.js and Strapi Starter Template to Vercel

### DISCUSSION THREAD

**Discussion by Decryptu** [vercel/community#6781] [Idea]
Hello,

Could you please add a starter template that combines Next.js with Strapi CMS to Vercel? This template would help developers quickly start projects using both technologies on your platform. Your feedback on this idea would be appreciated.

Thank you.

**Comment by amyegan** [comment]
Hey @Decryptu. We currently have the [Strapi Next.js Corporate Website Starter](https://vercel.com/templates/next.js/strapi-nextjs-corporate-starter) template available. And the [template submission form](https://vercel.com/templates/submit) is open for community contributions to the Templates Marketplace if you have a template you'd like to share

**Comment by Decryptu** [comment]
> Hey @Decryptu. We currently have the [Strapi Next.js Corporate Website Starter](https://vercel.com/templates/next.js/strapi-nextjs-corporate-starter) template available. And the [template submission form](https://vercel.com/templates/submit) is open for community contributions to the Templates Marketplace if you have a template you'd like to share

Ohhh my bad, its because I was only looking at the filters on the side, and Strapi was not in the list :
<img width="331" alt="Capture d’écran 2024-05-09 à 18 14 34" src="https://github.com/vercel/community/assets/90518536/bc212a84-be8f-48ce-b4fd-28c1789ab3d0">


Thank you for the link :)
